windows-driver-docs-pr/display/iddcx-versions.md

```diff
@@ -33,7 +34,7 @@ This page also lists the IddCx version (and **IddCxGetVersion** value) shipped w
 
 | Build number | Release codename | Release name         | IddCx version | IddCxGetVersion returned |
 | ------------ | ---------------- | -------------------- | ------------- | ------------------------ |
-| 22621        | n/a              | Windows 11 23H2      | 1.10          | 0x1A00 |
+| 22631        | n/a              | Windows 11 23H2      | 1.10          | 0x1A00 |
```

Comment:
What is special about build 22631 for IddCx such that you are adding it to this table, especially if it's not significant for Germanium (0x1A80)? If anything, it seems this should say that 22621 and 22631 are both associated with Windows 11 23H2 (I don't think 22621 was for Win 11 22H2).

Reply:
On further investigation, I see that 22631 was the build used for 23H2 official release, whereas latest WDK release used 22621. But you're referring to OS release in this table, so 22631 is correct. Sorry for confusion!